irohad/torii/impl/command_service.cpp

```diff
@@ -69,6 +64,16 @@ namespace torii {
     });
   }
 
+  namespace {
+    iroha::protocol::ToriiResponse makeResponse(
```

Comment:
Add documentation for the method.

Reply:
Isn't it self-explanatory?

Reply:
But the purpose is hidden.

Reply:
anonymous namespace make a clue that it's for internal usage
And there's nothing fancy, it just make response with the passed params